pyproject.toml

```
@@ -98,6 +98,11 @@ doc = [
     "sphinxemoji==0.2.0",
     "sphinx-reredirects==0.1.2",
     "vtk==9.2.6",
+    "sphinx-toolbox",
+    "sphinxcontrib-asciinema",
+    "sphinxcontrib-websupport",
+    "typed-ast",
+    "pydata-sphinx-theme",
```

I dont think we need that many new extensions. Do we need even `sphinx-toolbox` and `sphinxcontrib-websupport`?

```suggestion
    "sphinx-toolbox",
    "sphinxcontrib-websupport",
```